css/custom.css

```diff
@@ -249,3 +249,8 @@ a:focus, a:hover {
     color: #fff !important;
     background: #e12b00 !important;
 }
+
+
+.responsive_icons {
+  margin-top: 5px;
```

Comment:
It would be better to use ```%``` instead of ```px``` for fixing responsiveness.

Reply:
I think we can have small margins constant.

Reply:
Yes, we really don't need responsiveness is such cases IMO. Also, for larger screens the margin won't really come into play. It is mainly for the case when the icons move to a new row.

Reply:
@akshatnitd  @simsausaurabh I have checked out your suggestion. I prefer to go with a constant 5 px because this will be a standard solution across all screen sizes. Although even when I gave I a margin of 1% the things worked fine but perhaps in small screens 1% may be too small. So, I went with the constant 5px.